src/index.js

```diff
@@ -46,47 +44,35 @@ module.exports = {
     'id-length': ['error', {
       exceptions: ['_', 'e', 'i']
     }],
+    'id-match': ['error', '^[_a-zA-Z][a-zA-Z0-9]*$|^[A-Z][A-Z0-9]*_[A-Z-0-9]+$'],
```

Comment:
This still doesn't seem to allow variables such as `READY` or `GREATER_THAN_ZERO` (still only accepts one underscore).

I see the block `[A-Z-0-9]+`. Is that `-` between the Z and the 0 meant to be a `_`, or is just a typo to be deleted?